lattice/util.py

```diff
@@ -4,3 +4,6 @@ def snake_style(s):
     #return ''.join(['_'+c.lower() if c.isupper() else c for c in s]).lstrip('_')
     a = re.compile('((?<=[a-z0-9])[A-Z]|(?!^)[A-Z](?=[a-z]))')
     return a.sub(r'_\1', s).lower()
+
+def bigladder_filename_style(s):
```

Comment:
This function name is more specific than its functionality. Maybe this should be called "hyphen_separated_lowercase"?